src/software/backend/simulation/physics/physics_world.cpp

```diff
@@ -0,0 +1,98 @@
+#include "physics_world.h"
+
+#include "software/backend/simulation/physics/physics_ball.h"
+#include "software/backend/simulation/physics/physics_field.h"
+#include "software/backend/simulation/physics/physics_robot.h"
+#include "software/util/time/duration.h"
+#include "software/util/time/timestamp.h"
+#include "software/world/world.h"
+
+PhysicsWorld::PhysicsWorld(const World& world)
+{
+    b2Vec2 gravity(0, 0);
+    physics_world           = std::make_shared<b2World>(gravity);
```

Comment:
Could we change this name to `b2_world`? The reason I'm asking is that I would've guessed that the type of `physics_world` is `PhysicsWorld`, which confused me a bit below until I double checked it. This way it also doesn't match up with the `physics_field` and `physics_ball` below, which both actually _are_ `Physics*` types.

Reply:
done